docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '3.7'
+
+services:
+  trac:
+    image: ghcr.io/django/code.djangoproject.com:latest
+    # Comment out to use the ghcr.io image:
+    build:
+      context: ./
+      dockerfile: Dockerfile
+    command: ["/venv/bin/tracd", "--port", "9000", "-s", "trac-env"]
+    volumes:
+      - ./trac-env/:/code/trac-env/
+    ports:
+      - 9000:9000
+    depends_on:
+      - db
+  db:
+    image: postgres:11.2-alpine
```

Comment:
Aren't we using PG 10 right now?

Reply:
Ah, yes. Good catch. This was a copy-paste from dp.com, so that is likely incorrect too.